perf/perf_watch_point.py.data/wptest.c

```diff
@@ -0,0 +1,56 @@
+#include <linux/module.h>
```

Comment:
can we have author signature also in these files

Reply:
ok

Reply:
yeah sure